SPECS-SIGNED/mstflint-driver-signed/mstflint-signed.spec

```
@@ -0,0 +1,32 @@
+%define mstflint_unsigned_name mstflint-kernel
+%global kver %(/bin/rpm -q --queryformat '%{RPMTAG_VERSION}-%{RPMTAG_RELEASE}' $(/bin/rpm -q --whatprovides kernel-devel))
+%global install_mod_dir lib/modules/%{kver}/extra/%{name}
+Summary:        Mellanox firmware burning tool
+Name:           %{mstflint_unsigned_name}-signed
+Version:        4.21.0
+Release:        4%{?dist}
+License:        Dual BSD/GPL
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Kernel
+URL:            https://github.com/Mellanox/%{name}
+Source0:        https://github.com/Mellanox/%{name}/releases/download/v%{version}-1/%{name}-%{version}-1.tar.gz
+Requires:       kernel >= 5.15.0 and kernel < 5.16.0
+Requires:       kernel-headers >= 5.15.0 and kernel-headers < 5.16.0
```

This is a limitation of our tools, where I believe they don't handle the `and`, so we use this instead:
```suggestion
Requires:       kernel >= 5.15.0
Requires:       kernel < 5.16.0
Requires:       kernel-headers >= 5.15.0
Requires:       kernel-headers < 5.16.0
```